src/sentry/migrations/0020_auto_20191125_1420.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2019-11-25 14:20
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from sentry import eventstore
+from sentry.models import EventAttachment
```

Comment:
There's a chance this will not work correctly.
We should use `apps.get_model()` instead.
https://stackoverflow.com/questions/37765307/why-use-apps-get-model-when-creating-a-data-migration

Reply:
Thanks, done.